docs/INTRO-mingw.md

```diff
@@ -0,0 +1,96 @@
+# Introduction to SDL with MinGW
+
+Without getting deep into the history, MinGW is a long running project that aims to bring gcc to Windows. That said, there's many distributions, versions, and forks floating around. We recommend installing [MSYS2](https://www.msys2.org/), as it's the easiest way to get a modern MinGW(-w64) with a package manager to help with dependency management. This would allow you to follow the MSYS2 section below. 
+
+Otherwise you'll want to follow the "Other Distributions" section below.
+
+We'll start by creating a simple project to build and run [hello.c](hello.c).
+
+# MSYS2
+
+Open the `MSYS2 UCRT64` prompt and then ensure you've installed the following packages. This will get you working toolchain, CMake, Ninja, and of course SDL3.
+
+```sh
+pacman -S mingw-w64-ucrt-x86_64-gcc mingw-w64-ucrt-x86_64-ninja mingw-w64-ucrt-x86_64-cmake mingw-w64-ucrt-x86_64-sdl3
+```
+
+## Create the file CMakeLists.txt
+```cmake
+project(sdl_test C CXX)
+cmake_minimum_required(VERSION 3.26)
+
+find_package(SDL3 REQUIRED)
+
+add_executable(sdl_test)
+
+target_sources(sdl_test
+PRIVATE
+    hello.c
+)
+
+target_link_libraries(sdl_test SDL3::SDL3)
+```
+
+## Configure and Build:
+```sh
+cmake -S . -B build
+cmake --build build
+```
+
+## Run:
+
+### Linux/Mac/BSD/Windows (Non Visual Studio  Generator)/etc
+The executable is in the `build` directory:
+```sh
+cd build
+./hello
+```
+
+# Other Distributions
+
+Things can get quite complicated with other distributions of MinGW. If you can't follow [the cmake intro](INTRO-cmake.md), perhaps due to issues getting cmake to understand your toolchain, we can provide a guide that should get you up and running.
```

Comment:
We can? Is it this guide?

Reply:
Yeah, I'll rephrase this. Generally I think that if you have MinGW and a compatible CMake, the CMake guide is the way to go. But we do end up with people who insist on using their toolchain, so I wanted to pave a path for them.